suite-native/intl/scripts/backportEnglish.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * This script takes latest en-US.json and creates new file messages.ts
+ *
+ * Problem ?
+ * - messages.ts is created by developers using "developers English" which might be miles away from proper English
```

Comment:
Nit: I’d rephrase this a bit, since some of the translations are also created by our Design and Product teams, so it might not be accurate. I’d suggest something like:

"messages.ts might not reflect the final version of the translations. The en-us.json file contains polished versions approved by our Product and Marketing teams.... When meaning ... "